c6021light/src/tasks/RoutingTask/XpressNetForwarder.cpp

```diff
@@ -0,0 +1,84 @@
+#include "tasks/RoutingTask/XpressNetForwarder.h"
+
+#include "RR32Can/Constants.h"
+#include "RR32Can/messages/S88Event.h"
+#include "RR32Can/messages/SystemMessage.h"
+#include "RR32Can/messages/TurnoutPacket.h"
+
+#include "XpressNet/XpressNetMaster.h"
+
+namespace tasks {
+namespace RoutingTask {
+
+void XpressNetForwarder::forward(const RR32Can::CanFrame& frame) {
+  switch (frame.id.getCommand()) {
+    case RR32Can::Command::ACCESSORY_SWITCH: {
+      const RR32Can::TurnoutPacket turnoutPacket(const_cast<RR32Can::Data&>(frame.data));
+      if (!frame.id.isResponse()) {
+        // Send to XpressNet
+        // convert CAN turnout direction
+        uint8_t xn_direction;
+        if (turnoutPacket.getDirection() == RR32Can::TurnoutDirection::GREEN)
```

Comment:
Blöcke in for/while/if/else bitte immer mit {} einfassen!

Reply:
added everywhere